### PR TITLE
Fixed issue #9852

### PR DESCRIPTION
**Pull request by shivendra-webkul**
## Issue Reference
Fixed issue #9852
Create new address request for admin package

### CODE REVIEW

**Comment by ashishkumar-webkul** [comment]
@shivendra-webkul 
I have checked the PR with almost all locale and now it is working fine. As now no validation is appearing and it is accepting the desired locale translation which is required.

Please check the video for the reference.

**Video**

https://github.com/bagisto/bagisto/assets/92721837/d76587ff-6254-44a7-ad91-aa06dfd2ed4c

File: packages/Webkul/Admin/src/Http/Requests/AddressRequest.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Webkul\admin\Http\Requests;
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Http/Requests/AddressRequest.php:3]: Check namespace.

File: packages/Webkul/Admin/src/Http/Requests/CartAddressRequest.php [hunk 2]
```diff
@@ -47,14 +46,14 @@ public function rules(): array
...
             "{$addressType}.address"    => ['required', 'array', 'min:1'],
-            "{$addressType}.city"       => ['required'],
-            "{$addressType}.country"    => [new AlphaNumericSpace],
-            "{$addressType}.state"      => [new AlphaNumericSpace],
-            "{$addressType}.postcode"   => ['numeric'],
+            "{$addressType}.city"       => ['required', 'string'],
+            "{$addressType}.country"    => ['required', 'string'],
+            "{$addressType}.state"      => ['required', 'string'],
+            "{$addressType}.postcode"   => ['required', 'numeric'],
             "{$addressType}.phone"      => ['required', new PhoneNumber],
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/Http/Requests/CartAddressRequest.php:57]: Remove string.

File: packages/Webkul/Shop/src/Http/Requests/CartAddressRequest.php [hunk 3]
```diff
@@ -47,14 +46,14 @@ public function rules(): array
...
+            "{$addressType}.last_name"  => ['required', 'string'],
             "{$addressType}.email"      => ['required'],
             "{$addressType}.address"    => ['required', 'array', 'min:1'],
             "{$addressType}.city"       => ['required'],
-            "{$addressType}.country"    => [new AlphaNumericSpace],
-            "{$addressType}.state"      => [new AlphaNumericSpace],
-            "{$addressType}.postcode"   => ['numeric'],
+            "{$addressType}.country"    => core()->isCountryRequired() ? ['required', 'string'] : ['string'],
+            "{$addressType}.state"      => core()->isStateRequired() ? ['required', 'string'] : ['string'],
+            "{$addressType}.postcode"   => core()->isPostCodeRequired() ? ['required', 'numeric'] : ['numeric'],
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Http/Requests/CartAddressRequest.php:56]: Remove string.

File: packages/Webkul/Shop/src/Http/Requests/Customer/AddressRequest.php [hunk 4]
```diff
@@ -27,12 +26,12 @@ public function authorize()
...
+            'company_name' => ['string'],
+            'first_name'   => ['required', 'string'],
+            'last_name'    => ['required', 'string'],
             'address'      => ['required', 'array', 'min:1'],
-            'country'      => core()->isCountryRequired() ? ['required', new AlphaNumericSpace] : [new AlphaNumericSpace],
-            'state'        => core()->isStateRequired() ? ['required', new AlphaNumericSpace] : [new AlphaNumericSpace],
+            'country'      => core()->isCountryRequired() ? ['required', 'string'] : ['string'],
+            'state'        => core()->isStateRequired() ? ['required', 'string'] : ['string'],
             'city'         => ['required', 'string'],
             'postcode'     => core()->isPostCodeRequired() ? ['required', 'numeric'] : ['numeric'],
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Http/Requests/Customer/AddressRequest.php:36]: Remove string.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Shop/src/Http/Requests/Customer/AddressRequest.php (hunk 4 above)
Comment (shivendra-webkul) [reply packages/Webkul/Shop/src/Http/Requests/Customer/AddressRequest.php]: Done

File: packages/Webkul/Shop/src/Http/Requests/CartAddressRequest.php (hunk 3 above)
Comment (shivendra-webkul) [reply packages/Webkul/Shop/src/Http/Requests/CartAddressRequest.php]: Done

File: packages/Webkul/Admin/src/Http/Requests/CartAddressRequest.php (hunk 2 above)
Comment (shivendra-webkul) [reply packages/Webkul/Admin/src/Http/Requests/CartAddressRequest.php]: Done

File: packages/Webkul/Admin/src/Http/Requests/AddressRequest.php (hunk 1 above)
Comment (shivendra-webkul) [reply packages/Webkul/Admin/src/Http/Requests/AddressRequest.php]: Done

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**